packages/mobile/package.json

```diff
@@ -49,12 +49,14 @@
   "dependencies": {
     "@celo/client": "691d471",
     "@celo/react-components": "1.0.0",
-    "@celo/react-native-sms-retriever": "git+https://github.com/celo-org/react-native-sms-retriever#d3a2fdb",
+    "@celo/react-native-sms-retriever": "git+https://github.com/celo-org/react-native-sms-retriever#b88e502",
     "@celo/utils": "^0.1.1",
     "@celo/walletkit": "^0.0.14",
-    "@react-native-community/netinfo": "^2.0.4",
-    "@segment/analytics-react-native": "^1.1.0-beta.1",
-    "@segment/analytics-react-native-firebase": "^1.1.0-beta.1",
+    "@react-native-community/async-storage": "^1.6.2",
```

Comment:
I basically had to update all our RN deps. This will probably break some things, so keep any eye out, although hopefully it doesn't